<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fallout NV Pip Colorizer</string>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>2</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Colór Quieres</string>
   </property>
  </widget>
  <widget class="QPushButton" name="picker">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>21</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Recoja</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>48</y>
     <width>82</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Colór de Fallout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="convert">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>65</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Converta</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="falloutColor">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>66</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="desiredColor">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>22</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="write">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>93</y>
     <width>103</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ahorre/Carge</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gtfo">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>93</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Salga</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
